lychee-bin/src/stats.rs

```diff
@@ -42,6 +42,10 @@ impl ResponseStats {
 
     pub(crate) fn add(&mut self, response: Response) {
         let Response(source, ResponseBody { ref status, .. }) = response;
+        if status.is_unsupported() {
+            // Silently skip unsupported URIs for now
```

Comment:
You can add a TODO: or FIXME: keyword

Reply:
Maybe I should remove the "for now" part, because I think it's a permanent thing. If we don't support a URI, we just ignore it.

Reply:
right.

So we currently support HTTP(s) and MAILTO? FILE?

It's nice to add them to readme.

Reply:
yup